File: NgCmsApi/Models/ContentModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using System.Collections.Generic;

namespace NgCmsApi.Models
{
    public class ContentModel
    {
        [Required]
        [JsonProperty("guid")]
        public Guid Guid { get; set; }

        [Required]
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("content")]
        public string Content { get; set; }
    }
}

File: NgCmsWeb/src/app/models/content.model.ts
import {Serializable} from './../interfaces/serializable.interface';

export class ContentModel implements Serializable<ContentModel> {
    guid: string;
    name: string;
    content: string;

    deserialize(input) {
        this.guid = input.guid;
        this.name = input.name;
        this.content = input.content;

        return this;
    }
}
